.gitignore

```diff
@@ -103,3 +103,6 @@ dist
 # TernJS port file
 .tern-port
 .DS_Store
+
+# Local History
+.history
```

Comment:
I remember it's an old ignore that I used to use. Are you sure it's still needed here?

Reply:
It is from ur previous changes on this PR, i can delete it if not needed